index.html

```diff
@@ -125,6 +125,10 @@ <h3>Other tools</h3>
                         Amplitude <span class="fa fa-lock"></span>
                         <div class="dashboard-description">Dive into product analytics.</div>
                     </a>
+                    <a class="list-group-item" href="https://www.mozdatacollective.com/">
```

Comment:
Maybe bump it up a position so that it's adjacent to "Reports Portal"? They are confusingly similar, RTMO and MDC, so they may as well be nearby to maximize confusion :)

Reply:
Thanks. I bumped it up to maximize confusion. :)